src/solution.c

```diff
@@ -396,6 +401,53 @@ static void sol_thd_sleep(systime_t *deadline, systime_t interval)
   chSysUnlock();
 }
 
+/** Calculates Iono and Tropo correction
+ * \param n_ready_tdcp The number of measurements written to `m_tdcp`
+ * \param nav_meas_tdcp Pointer to measurements array */
+void calc_iono_tropo(u8 n_ready_tdcp,
```

Comment:
Can you also call this in base_obs.c so our base positions are also improved:
Just before here https://github.com/swift-nav/piksi_firmware/blob/master/src/base_obs.c#L189